### PR TITLE
extracted common code into TestContext

### PR DESCRIPTION
**Pull request by vladimir32pa**

### CODE REVIEW

File: app/components/view-question-title/macro.test.js [hunk 1]
```diff
@@ -1,26 +1,9 @@
 import request from 'supertest';
-import express from 'express';
-import nunjucks from 'nunjucks';
 import cheerio from 'cheerio';
-import { App } from '../../../app';
+import { TetsContext } from '../../testContext/testContext'
```
Comment (alibombaye) [inline app/components/view-question-title/macro.test.js]: a typo here `TetsContext `

File: app/components/view-question-title/macro.test.js [hunk 2]
```diff
@@ -31,7 +14,7 @@ describe('view-question-title', () => {
       },
     };
 
-    const dummyApp = createDummyApp(context);
+    const dummyApp = TetsContext(macroWrapper, context);
```
Comment (alibombaye) [inline app/components/view-question-title/macro.test.js]: Nice 👍

File: app/testContext/testContext.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+import nunjucks from 'nunjucks';
+import { App } from '../../app';
+
+export const TetsContext = (macroWrapper, componentContext) => {
```
Comment (alibombaye) [inline app/testContext/testContext.js]: Just a little unsure about the name. I would make it little more clear about the purpose of this function. So something like `createTestHarness`.

What do you think?

File: app/testContext/testContext.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import express from 'express';
```
Comment (alibombaye) [inline app/testContext/testContext.js]: Again not to keen on the name of the file. I would again be bit more specific and name the directory `testUtils` and the file `helpers.js`

File: app/components/view-question-title/macro.test.js (hunk 1 above)
Comment (vladimir32pa) [reply app/components/view-question-title/macro.test.js]: fixed

File: app/testContext/testContext.js (hunk 3 above)
Comment (kat1906) [reply app/testContext/testContext.js]: or createDummyApp still as it hasn't changed in function

File: app/testContext/testContext.js (hunk 4 above)
Comment (vladimir32pa) [reply app/testContext/testContext.js]: can rename directory to `testUtils`  but would keep testContext in its own file and put other little helpers in a `helpers.js` if we have any in future

**Comment by alibombaye** [comment]
Yeah that looks good. Go ahead with this for the rest 👍

**Pull request merged**